articles/openshift/concepts-restrict-egress-traffic.md

```diff
@@ -0,0 +1,59 @@
+
+> [!IMPORTANT]
+> The specific network rules, endpoints, ports, and other information provided in this document is subject to change.
+
+It contains the cluster requirements for a base AKS deployment, and additional requirements for optional addons and features. [An example will be provided at the end on how to configure these requirements with Azure Firewall](#restrict-egress-traffic-using-azure-firewall). However, you can apply this information to any outbound restriction method or appliance.
+
+## Background
+s
+Azure Red Hat OpenShift 4 clusters are deployed on two virtual networks. You create these networks as part of creating your cluster, and they have  **outbound** dependencies on services outside of the virtual networks..
+
+For management and operational purposes, nodes in an Azure Red Hat OpenShift 4 cluster need to access certain ports and fully qualified domain names (FQDNs). These endpoints are required for the nodes to communicate with the API server, or to download and install core Kubernetes cluster components and node security updates. For example, the cluster needs to pull base system container images from Microsoft Container Registry (MCR).
+
+The Azure Red Hat OpenShift 4 outbound dependencies are almost entirely defined with FQDNs, which do not have static addresses behind them. The lack of static addresses means that Network Security Groups can not be used to lock down the outbound traffic from an Azure Red Hat OpenShift 4 cluster. 
+
+By default, Azure Red Hat OpenShift 4 clusters have unrestricted outbound (egress) internet access. This level of network access allows nodes and services you run to access external resources as needed. If you wish to restrict egress traffic, a limited number of ports and addresses must be accessible to maintain healthy cluster maintenance tasks. The simplest solution to securing outbound addresses lies in use of a firewall device that can control outbound traffic based on domain names. Azure Firewall, for example, can restrict outbound HTTP and HTTPS traffic based on the FQDN of the destination. You can also configure your preferred firewall and security rules to allow these required ports and addresses.
+
+> [!IMPORTANT]
+> This document covers only how to lock down the traffic leaving the Azure Red Hat OpenShift 4 subnet. Azure Red Hat OpenShift 4 has no ingress requirements by default.  Blocking **internal subnet traffic** using network security groups (NSGs) and firewalls is not supported. To control and block the traffic within the cluster, use [***Network Policies***][network-policy].
+
+## Required outbound network rules and FQDNs
+
+The following network and FQDN/application rules are required for an Azure Red Hat OpenShift 4 cluster. You can use them if you wish to configure a solution other than Azure Firewall.
+
+* IP Address dependencies are for non-HTTP/S traffic (both TCP and UDP traffic)
```

Comment:
Might be able to remove this line as we don't have and non-HTTP/S traffic at this time.